examples/apps/cchmc_ped_abd_ct_seg_app/scripts/map_build.sh

```
@@ -12,18 +12,19 @@
 image_version=$2
-sdk_version=$3
+monai_deploy_app_sdk_version=$3
+cuda_version=$4
 
 # load in environment variables
 source .env
 
-# build MAP 
-monai-deploy package cchmc_ped_abd_ct_seg_app -m $HOLOSCAN_MODEL_PATH -c cchmc_ped_abd_ct_seg_app/app.yaml -t ${tag_prefix}:${image_version} --platform x86_64 --sdk-version ${sdk_version} -l DEBUG
+# build MAP - let pacakger choose base image
+monai-deploy package ../cchmc_ped_abd_ct_seg_app -m $HOLOSCAN_MODEL_PATH -c ../cchmc_ped_abd_ct_seg_app/app.yaml -t ${tag_prefix}:${image_version} --platform x86_64 --sdk-version ${monai_deploy_app_sdk_version} --cuda ${cuda_version} -l DEBUG
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)